### DISCUSSION TITLE
Hexo-基于 CryptoJS-AES 的 Hexo 文章片段加密插件 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#333] [Announcements]
# Hexo-基于 CryptoJS-AES 的 Hexo 文章片段加密插件 | Zi-Zi's Journey

小小怪说 AES 算法很安全！

https://gz-metal-cell.github.io/2024/05/23/Hexo-%E5%9F%BA%E4%BA%8E%20CryptoJS-AES%20%E7%9A%84%20Hexo%20%E6%96%87%E7%AB%A0%E7%89%87%E6%AE%B5%E5%8A%A0%E5%AF%86%E6%8F%92%E4%BB%B6/

**Comment by promefire** [comment]
太强了/(ㄒoㄒ)/~~

**Reply by GZ-Metal-Cell** [reply]
😭有了这个算法😭我就可以在 Diary 里偷偷说 pro 哥坏话了😭pro 哥现在开阅读模式也没有用了😭
